docs/modules/ROOT/pages/how-tos/upgrade-mimir-v1-v2.adoc

```diff
@@ -0,0 +1,237 @@
+= Upgrade component version v1 to v2
+
+When migrating from component version v1 to v2, the following breaking changes require manual setup of zone aware replication on existing instances:
+
+* https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md#400[mimir-distributed] enables zone aware replication for ingesters by default. An upgrade without manual migration will incur data loss.
+
+
+== Prerequisites
+
+You need to be able to execute the following CLI tools locally:
+
+* `yq` https://github.com/mikefarah/yq[yq YAML processor] (Version 4 or newer)
+
+== Setup environment
+
+. Access to API
++
+[source,bash]
+----
+# For example: https://api.syn.vshn.net
+# IMPORTANT: do NOT add a trailing `/`. Commands below will fail.
+export COMMODORE_API_URL=<lieutenant-api-endpoint>
+
+export CLUSTER_ID=<lieutenant-cluster-id> # Looks like: c-<something>
+export TENANT_ID=$(curl -sH "Authorization: Bearer $(commodore fetch-token)" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+
+# Parameter key under which your mimir component instance is configured in the tenant repo
+export MIMIR_PARAM_KEY=<mimir_param_key>
+----
+
+== Upgrade the component without zone aware replication
+
+. Compile the cluster catalog to create a local working directory
++
+[source,bash]
+----
+commodore catalog compile "${CLUSTER_ID}"
+----
+
+. Update the component version to v2.x in your cluster configuration:
++
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.components.mimir.version = 2.0.0" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
+
+. Add temporary migration configuration:
++
+[source,bash]
+----
+cd inventory/classes/$TENANT_ID
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.alertmanager.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.ingester.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.store_gateway.zoneAwareReplication.enabled = false" ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.${MIMIR_PARAM_KEY}.helm_values.rollout_operator.enabled = false" ${CLUSTER_ID}.yml
+
+cd ../../..
+----
++
+This configuration disables zone aware replication. In the first step, the new component version is deployed with this setting disabled.
+
+. Compile and push the cluster catalog
```

Comment:
Somewhere before this step, you're missing a `git commit -am'Update mimir' && git push origin master` in the tenant repo